src/pages/AppBasicAdminPage/AppBasicAdminCard.tsx

```diff
@@ -70,7 +70,9 @@ const AppBasicAdminCard: React.VFC<AppBasicAdminCardProps> = ({ ...cardProps })
             {uploading ? 'Uploading' : 'Start Upload'}
           </Button>
         </div>
-        <img src={settings['logo']} alt="logo" height={64} />
+        <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', maxWidth: 320 }}>
```

Comment:
why use maxWidth?
just curious, don't mind

Reply:
maintain image layout ?

Reply:
`object-fit: contain` will be better ?